{"author":"dotnet-bot","ms.author":"dotnetcontent","manager":"wpickett","breadcrumb_path":"/dotnet/toc.json","ms.topic":"managed-reference","ms.prod":".net","search.ms_sitename":"Docs","search.ms_docsetname":"docs-internal","version":0,"locale":"en-us","site_name":"Docs","search.ms_product":"MSDN","depot_name":"MSDN.docs-internal","original_ref_skeleton_git_url":"https://github.com/dotnet/docs-internal/blob/master/api/System.DirectoryServices.ActiveDirectory.ActiveDirectoryOperationException.yml","open_to_public_contributors":false,"api_name":["System.DirectoryServices.ActiveDirectory.ActiveDirectoryOperationException","System.DirectoryServices.ActiveDirectory.ActiveDirectoryOperationException..ctor","System.DirectoryServices.ActiveDirectory.ActiveDirectoryOperationException.ErrorCode","System.DirectoryServices.ActiveDirectory.ActiveDirectoryOperationException.GetObjectData"],"api_location":["System.DirectoryServices.dll"],"topic_type":["apiref"],"api_type":["Assembly"],"f1_keywords":["System.DirectoryServices.ActiveDirectory.ActiveDirectoryOperationException","System::DirectoryServices::ActiveDirectory::ActiveDirectoryOperationException","System.DirectoryServices.ActiveDirectory.ActiveDirectoryOperationException.#ctor","System::DirectoryServices::ActiveDirectory::ActiveDirectoryOperationException::#ctor","System.DirectoryServices.ActiveDirectory.ActiveDirectoryOperationException.ErrorCode","System::DirectoryServices::ActiveDirectory::ActiveDirectoryOperationException::ErrorCode","System.DirectoryServices.ActiveDirectory.ActiveDirectoryOperationException.GetObjectData","System::DirectoryServices::ActiveDirectory::ActiveDirectoryOperationException::GetObjectData"],"dev_langs":["csharp"],"updated_at":"2017-03-28 03:03 AM","document_id":"5f47e268-bcc9-64f3-3bd5-8a21e7df81e1","content_git_url":"https://github.com/dotnet/docs/blob/master/xml/System.DirectoryServices.ActiveDirectory/ActiveDirectoryOperationException.xml","internalonly":false,"layout":"Reference","pagetype":"Reference","title":"ActiveDirectoryOperationException class | Microsoft Docs","description":"The ActiveDirectoryOperationException class exception is thrown when an underlying directory operation fails.\n","toc_asset_id":"api/_splitted/System.DirectoryServices.ActiveDirectory/toc.json","toc_rel":"_splitted/System.DirectoryServices.ActiveDirectory/toc.json","source_url":"","ms.assetid":"System.DirectoryServices.ActiveDirectory.ActiveDirectoryOperationException","canonical_url":"https://ppe.docs.microsoft.com/en-us/dotnet/api/system.directoryservices.activedirectory.activedirectoryoperationexception","is_dynamic_rendering":true,"content_type":"application/json","publish_version":"{\"opbuild.templates.docs\":\"1.30.0\",\"commit_id\":\"f3885c86bfdf93482927f938e72eb48f1b16b70e\",\"branch\":\"master\",\"opbuild.scripts\":\"1.30.0\",\"opbuild.htmltopdftool\":\"1.30.0\",\"opbuild.gitcontributorinformationresolver\":\"1.30.0\",\"opbuild_service\":\"1.30.128.0\",\"docfx.console\":\"2.15.0\",\"opbuild.resolvedependencyconsole\":\"1.30.0\",\"opbuild.schemavalidationconsole\":\"1.30.0\",\"opbuild.redirectionresolver\":\"1.30.0\",\"opbuild.templates.intellisense\":\"1.30.0\",\"dependencies\":[{\"branch\":\"preview\",\"path\":\"_themes\",\"url\":\"https://github.com/Microsoft/templates.docs.msft\",\"commit_id\":\"a2590cfa118ad6a09d77514a947400c884574af6\",\"opbuild.templates.common\":\"1.30.0\"},{\"branch\":\"master\",\"path\":\"_themes.pdf\",\"url\":\"https://github.com/Microsoft/templates.docs.msft.pdf\",\"commit_id\":\"ed2aa9642124365e156c2670dab210e368095ebd\",\"opbuild.templates.common\":\"1.30.0\"}]}"}